lms/resources/_js_config/file_picker_config.py

```diff
@@ -105,10 +105,10 @@ def microsoft_onedrive(cls, request, application_instance):
         }
 
     @classmethod
-    def vital_source_config(cls, _request, application_instance):
-        """Get Vital Source config."""
-        enabled = application_instance.settings.get("vitalsource", "enabled", False)
-        return {"enabled": enabled}
+    def vitalsource_config(cls, request, _application_instance):
+        """Get VitalSource config."""
```

Comment:
The name of this method was wrong as the product is "VitalSource" rather than "Vital Source". Thought I'd fix it while I was here.